wiki/heath_potion.md

```diff
@@ -0,0 +1,26 @@
+Turrets 
+=======
+
+* Icon: `♥`
+* Buy with: `l, 1, *`
+* Use with: `u, 1, *`
+* Cost: `50 Ore`
```

Comment:
Use this following formatting instead. The below values are placeholders.

* Cost:
    * Construction Cost: 1
    * Buy from Pharmacy with `M` standing: 5
    * Buy from Pharmacy with `A` standing: 10
    * Buy from Pharmacy with `N` standing: 15
    * Buy from Pharmacy with `E` standing: 20